### DISCUSSION TITLE
Scrolling in side nav

### DISCUSSION THREAD

**Discussion by joppehoekstra** [requarks/wiki#3647] [Error / Bug Report]
### Actual behavior
When clicking the hamburger icon in the bottom left on an iPhone, you cannot scroll down in the side navigation if the list of links does not fit the screen.

### Expected behavior
You should be able to scroll.

### Steps to reproduce the behavior
See description for actual behavior. Strangely enough, this does not happen on the documentation site (but it does happen with my instance https://activisthandbook.org)

**Comment by D1reW0lf** [comment]
I think I’m having the same problem on iOS Safari (latest version as of this writing)
I can’t select the last option of the menu (Z).
On IPad it does work fine. 

![IMG_1028](https://github.com/requarks/wiki/assets/1677707/c9e1d8d6-bb6b-4ff5-a2c3-6965d05333b6)
